js/tc39/breaking_test_errors.json

```diff
@@ -2217,7 +2212,8 @@
   "test/language/expressions/new/spread-sngl-obj-ident.js-strict:true": "test/language/expressions/new/spread-sngl-obj-ident.js: SyntaxError: test/language/expressions/new/spread-sngl-obj-ident.js: Unexpected token (56:3)\n  54 |   });\n  55 |   callCount += 1;\n\u003e 56 | }({...o});\n     |    ^\n  57 | \n  58 | assert.sameValue(callCount, 1);\n  59 |  at \u003ceval\u003e:2:28542(114)",
   "test/language/expressions/object/__proto__-duplicate-computed.js-strict:true": "test/language/expressions/object/__proto__-duplicate-computed.js: Test262Error: prototype is defined Expected SameValue(«[object Object]», «[object Object]») to be true at harness/sta.js:22:9(49)",
   "test/language/expressions/object/__proto__-duplicate.js-strict:true": "test/language/expressions/object/__proto__-duplicate.js: error is not an object (Test262: This statement should not be evaluated.)",
-  "test/language/expressions/object/__proto__-permitted-dup-shorthand.js-strict:true": "test/language/expressions/object/__proto__-permitted-dup-shorthand.js: SyntaxError: test/language/expressions/object/__proto__-permitted-dup-shorthand.js: Redefinition of __proto__ property (22:2)\n  20 | var obj = {\n  21 |   __proto__,\n\u003e 22 |   __proto__,\n     |   ^\n  23 | };\n  24 | \n  25 | assert(obj.hasOwnProperty(\"__proto__\")); at \u003ceval\u003e:2:28542(114)",
+  "test/language/expressions/object/__proto__-fn-name.js-strict:true": "test/language/expressions/object/__proto__-fn-name.js: Test262Error: Expected true but got false at harness/sta.js:22:9(49)",
+  "test/language/expressions/object/__proto__-permitted-dup-shorthand.js-strict:true": "test/language/expressions/object/__proto__-permitted-dup-shorthand.js: TypeError: Object prototype may only be an Object or null: 2 at test/language/expressions/object/__proto__-permitted-dup-shorthand.js:21:12(9)",
```

Comment:
AnnexB handling of `__proto__` property where the name of the property is wrong (it is now "__proto__" but it shouldn't) - of no relevance IMO 

also a lot of the other AnnexB tests about `__proto__` are failing, so support for this seems ... not great